webapp/src/components/DatasetWarnings/DatasetDistribution.tsx

```diff
@@ -4,6 +4,22 @@ import MetricsDataGrid from "components/DatasetWarnings/MetricsDataGrid";
 import { ResponsivePlotWrapper, WarningPlot } from "components/PlotWrapper";
 import React from "react";
 import { DatasetWarningGroup } from "types/api";
+import Description from "components/Description";
+
+interface DescriptionMap {
+  [key: string]: string;
+}
+
+const WARNING_GROUP_DESCRIPTION: DescriptionMap = {
+  GeneralWarnings: "See class distributions differences. ",
+  SyntacticWarnings:
+    "Review the arrangement of utterances between your training and testing data. ",
```

Comment:
I would add the spaces in the keys here
```suggestion
  "General Warnings": "See class distributions differences. ",
  "Syntactic Warnings":
    "Review the arrangement of utterances between your training and testing data. ",
```
to avoid the `.replace(/\s/g, "")`
```diff
                 <Description
-                  text={
-                    WARNING_GROUP_DESCRIPTION[
-                      warningGroup.name.replace(/\s/g, "")
-                    ]
-                  }
-                  link={
-                    WARNING_GROUP_DOC_LINK[warningGroup.name.replace(/\s/g, "")]
-                  }
+                  text={WARNING_GROUP_DESCRIPTION[warningGroup.name]}
+                  link={WARNING_GROUP_DOC_LINK[warningGroup.name]}
                 />
```